src/views/Settings.vue

```diff
@@ -123,21 +125,23 @@ export default defineComponent({
       });
       return popover.present();
     },
+    setFacility (facility: any) {
+      if (this.userProfile)
+        this.store.dispatch('user/setFacility', {
+          'facility': this.userProfile.facilities.find((fac: any) => fac.facilityId == facility['detail'].value)
+        });
+    },
     logout () {
       this.store.dispatch('user/logout').then(() => {
-        this.store.dispatch('picklist/clearPicklist')
```

Comment:
I think this is not related. We should revert.

Reply:
Reverted this change.